sdk/src/wallet/message_interface/account_method.rs

```diff
@@ -254,20 +254,22 @@ pub enum AccountMethod {
     /// [`MinimumRequiredStorageDeposit`](crate::wallet::message_interface::Response::MinimumRequiredStorageDeposit)
     MinimumRequiredStorageDeposit { output: OutputDto },
     /// Mint more native token.
-    /// Expected response: [`MintTokenTransaction`](crate::wallet::message_interface::Response::MintTokenTransaction)
+    /// Expected response:
+    /// [`MintNativeTokenTransaction`](crate::wallet::message_interface::Response::MintNativeTokenTransaction)
     #[serde(rename_all = "camelCase")]
-    IncreaseNativeTokenSupply {
+    MintNativeToken {
         /// Native token id
         token_id: TokenId,
         /// To be minted amount
         mint_amount: U256,
         options: Option<TransactionOptionsDto>,
     },
-    /// Mint native token.
-    /// Expected response: [`MintTokenTransaction`](crate::wallet::message_interface::Response::MintTokenTransaction)
+    /// Create a native token.
+    /// Expected response:
+    /// [`MintNativeTokenTransaction`](crate::wallet::message_interface::Response::MintNativeTokenTransaction)
```

Comment:
Also here
```suggestion
    /// [`CreateNativeTokenTransaction`](crate::wallet::message_interface::Response::CreateNativeTokenTransaction)
```

Reply:
As said above a mint happens for create and for mint.